### DISCUSSION TITLE
RuneScape Tree Run Helper.

### DISCUSSION THREAD

**Discussion by Echo-Hollow** [runelite/runelite#17196] [Ideas]
Embark on a fruitful adventure with our comprehensive guide for a complete tree run in RuneScape. To ensure a seamless experience, gather the following items:

Essential Tools:

Bottomless Bucket
Spade
Seed Dibber
Coin(s)

Required Saplings:
[Example of selected sapling(s)]
5 Palm Saplings
6 Magic/Yew Saplings

Payment Items:

Noted Papaya Payment
Noted Cactus Spine Payment

Teleportation Items:

Taverly Teleport Tab
Lumbridge Teleport Tab
Varrock Teleport Tab
House Teleport Tab(s)
Ring of Wealth
Brimhaven Teleport Tab
Camelot Teleport Tab
Teleport Crystal (Charged)
Royal Seed Pod
Varrock Spirit Tree

Additional Notes:

Palm + Yew = Gnome Stronghold
Palm = Gnome Maze
Palm = Lletya
Palm = Brimhaven
Palm = Catherby Beach
Yew = Falador Park
Yew = Varrock Garden
Yew = Lumbridge (Behind Castle)
Yew = Taverly (Near Gate/Lake)
Yew = Farming Guild 
Palm = Farming Guild [85 Farming Required]

Special Location for Hespori:

Hespori resides in the Farming Guild.

Growth Time: 22-32 hours.

Recommended Strategy: Use avas instead of a cape and intermittently blowpipe/dart the four outer plants, using melee/wiki-best-in-slot until defeated.

🌳🌟

Prepare for a bountiful tree-run adventure, maximizing your farming gains in RuneScape!

**Comment by Felanbird** [comment]
This should be added to https://github.com/Zoinkwiz/quest-helper as the plugin already has a guide for herb runs.
![image](https://github.com/runelite/runelite/assets/41973452/d16f1200-00b2-420a-9139-4291a9ea1aa2)

**Reply by Echo-Hollow** [reply]
'Lazy Farming' plugin. Have not tested it, yet!

**Comment by cconwaybhs** [comment]
Looks like this was eventually added to the Quest Helper plugin.
![image](https://github.com/user-attachments/assets/006b0f03-0c94-4adc-bbd2-c4fd715f49a6)
